<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>babelConnect</class>
 <widget class="QWidget" name="babelConnect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect -- Babel</string>
  </property>
  <widget class="QLabel" name="settingsLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="enterServerLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter Server IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="enterPortLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter Port:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPage">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enterPort">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="enterIP">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
